lib/rspec-puppet/example/function_example_group.rb

```diff
@@ -21,7 +21,7 @@ def execute(*args)
       end
 
       # compatibility alias for existing tests
-      def call(scope, *args)
```

Comment:
This is part of the public API, and cannot be changed, without accounting for existing callers.